iskolivery/app/views/users/_request.html.erb

```diff
@@ -36,6 +37,18 @@ This work is licensed under a Creative Commons Attribution-ShareAlike 4.0 Intern
 	<!-- </div> -->
 </div>
 
+<script>
+	$(document).ready(function(){
+	    // All your normal JS code goes in here
+	    $(".rating").rating();
+	});
+	$(document).ready(function(){
+		var rate = <%= @user.rating %>;
```

Comment:
this will return the rating for the user viewing the homepage, not the user that posted the request